db/models.py

```diff
@@ -38,7 +38,17 @@ class CircuitScheduleDoc(BaseModel):
 class TAGDoc(BaseModel):
     """TAG model"""
     tag_type: int
-    value: int
+    value: Union[int, str]
+
+    @validator('value')
+    def validate_value(cls, value):
+        """Validate value when is a string"""
+        if isinstance(value, int):
+            return value
+        if isinstance(value, str) and value in ("any", "untagged"):
+            return value
+        raise ValueError("value as string allows 'any', 'untagged' and the ",
+                         "format 'n/n'")
```

Comment:
Shouldn't we remove the `"n/n"` part from the message? and then leave for vlan range to take from this point forward?

Reply:
Yes, that is a leftover, removed.

Reply:
Got it. Let's close this thread then.